loc_books/books.py

```diff
@@ -0,0 +1,213 @@
+import datetime
+import functools
+import multiprocessing.dummy as mp
+import os
+import sys
+from pathlib import Path
+from time import time
+
+import click
+import pandas as pd
+import requests
+from furl import furl
+from pyrate_limiter import Duration, Limiter, RequestRate
+from requests_ratelimiter import LimiterSession
+from tenacity import (
+    RetryError,
+    Retrying,
+    retry,
+    stop_after_attempt,
+    wait_random_exponential,
+)
+from tqdm import tqdm
+
+from licensed_pile import logs
+from licensed_pile.licenses import PermissiveLicenses
+from licensed_pile.write import to_dolma
+
+data_path = Path(__file__).resolve().parent / "data"
+
+metadata_exports_path = data_path / "exports/metadata"
```

Comment:
Globals like this should be all upper case, e.g., `METADATA_EXPORTS_PATH`

Would it be hard to move this into the class by setting them during the `__init__`? Then you could pass in `data_path` to make it easy to configure.